va-gov/includes/header.html

```diff
@@ -10,7 +10,8 @@
 <meta http-equiv="X-UA-Compatible" content="IE=edge" />
 <meta name="HandheldFriendly" content="True">
 <meta name="MobileOptimized" content="320">
-<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+<meta name="viewport" content="width=device-width, initial-scale = 1.0, maximum-scale=1.0, user-scalable=no" />
```

Comment:
I think we were going to take this back out?

Reply:
I thought I removed this but I guess I didn't. I'll revert it back to the old one.

Reply:
I also fixed that border problem but it must have gotten reintroduced when I pulled from masters.